src/lib/gundb/gundb.native.js

```diff
@@ -1,7 +1,8 @@
 import Gun from 'gun/gun'
 import 'gun/sea'
-import 'gun-asyncstorage'
 import './gundb-extend'
+import 'gun-asyncstorage'
```

Comment:
You sure this is the way?

Doc says to import it at the app entry point https://github.com/staltz/gun-asyncstorage

Reply:
good question

Reply:
it should be ok.